src/test/kotlin/com/igorwojda/list/capitalizeFirst/challenge.kt

```diff
@@ -4,7 +4,12 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun capitalizeFirst(list: List<String>): List<String> {
-    TODO("not implemented")
+    return if (list.isEmpty()) emptyList() else listOf(list[0].capitalize()) + capitalizeFirst(
```

Comment:
Look at: https://kotlinlang.org/docs/collection-parts.html and simplify your code